dataplane/linux/bpf_ep_mgr.go

```diff
@@ -364,6 +368,12 @@ func (m *bpfEndpointManager) onInterfaceUpdate(update *ifaceUpdate) {
 	m.ifacesLock.Lock()
 	defer m.ifacesLock.Unlock()
 
+	if update.State == ifacemonitor.StateUnknown {
```

Comment:
IIUC, `StatetUnknown` is for cases when an interface does not have neither `StateUp` nor `StateDown`. This indicated for example when an interface is removed. In that case we should remove the file to clean up the directory.
For interfaces with `StateDown` we can keep the file since it won't change when the state changes to `StateUp`.
@fasaxc Have I got it right?

Reply:
Yes, "unknown" means "interface is gone" so I think it makes sense to forget the state then